src/components/Avatar.js

```diff
@@ -44,10 +44,16 @@ class Avatar extends PureComponent {
             this.props.size === 'small' ? styles.avatarSmall : styles.avatarNormal,
             ...this.props.imageStyles,
         ];
+
+        const roomAvatarStyle = [
+            this.props.size === 'small' ? styles.roomAvatarSmall : styles.roomAvatarNormal,
```

Comment:
@sobitneupane is there any reason `roomAvatarStyle` was created?
Can't we use `imageStyle`?

Reply:
If we use background color in roomAvatar as well, result looks like this (#admins room):-
<img width="350" alt="" src="https://user-images.githubusercontent.com/25876548/153997562-60ff16e7-44d4-40f6-8ecf-45bd17e2487e.png" />

We can achieve the same (required) result by using imageStyle but some changes should be made in imageStyle:-
```
        const imageStyle = [
            this.props.size === 'small' ? styles.avatarSmall : styles.avatarNormal,
            this.props.isChatRoom ? {} : {backgroundColor: themeColors.icon},
            ...this.props.imageStyles,
        ];
```

Reply:
> const imageStyle = [
            this.props.size === 'small' ? styles.avatarSmall : styles.avatarNormal,
            this.props.isChatRoom ? {} : {backgroundColor: themeColors.icon},
            ...this.props.imageStyles,
        ];

Ah thanks, let's do it this way then.

Reply:
Okay. Thanks.